src/libs/ContactUtils.ts

```diff
@@ -0,0 +1,101 @@
+import CONST from '@src/CONST';
+import type {PersonalDetails} from '@src/types/onyx';
+import type {DeviceContact, StringHolder} from './ContactImport/types';
+import * as OptionsListUtils from './OptionsListUtils';
+import {getAvatarForContact} from './RandomAvatarUtils';
+
+function sortEmailObjects(emails?: StringHolder[]): string[] {
```

Comment:
Could we minimalize this function?
1. Filter out null/undefined value and convert it to string array
2. Using the sort function in lodash to sort
Wdyt?

Reply:
Yeah, you're right, it's a bit cumbersome. I prefer using built-in sort methods over Lodash ones since they're a bit faster, though the difference is negligible
I've updated it, and now it looks more compact